### PR TITLE
Adds check and fix for str in tokenization process

### PR DESCRIPTION
**Pull request by SimonDegrafKern**
PR to fix gateway: https://github.com/code-kern-ai/refinery-gateway/pull/146
tokenizer: https://github.com/code-kern-ai/refinery-tokenizer/pull/37

Todo: https://www.notion.so/kern-ai/more-robust-tokenization-process-ea72185be75b453cac945838864d0a56

### CODE REVIEW

File: controller/tokenizer.py [hunk 1]
```diff
@@ -19,6 +19,10 @@ def get_doc_bin_in_bytes(
             if not to_be_tokenized:
                 # None / null types can't be tokenized by spacy so dummy string is used
                 to_be_tokenized = ""
+
+            if type(to_be_tokenized) != str:
```
Comment (lumburovskalina) [inline controller/tokenizer.py]: out of curiosity, can we check the type using the function `isInstance(to_be_tokenized,str)`?

Comment (SimonDegrafKern) [reply controller/tokenizer.py]: I never used a function called isInstance but seems that there is a similar built-in method provided by python, so most probably yes (btw. Pythonic would be is_instance, even tho the built-in function is called isinstance() what is quite confusing :D)

Comment (JWittmeyer) [reply controller/tokenizer.py]: isinstance is the preferred way for most cases

Comment (SimonDegrafKern) [reply controller/tokenizer.py]: changed to use isinstance

**Review by JWittmeyer — APPROVED** [review]

**Pull request merged**